### ISSUE TITLE
[REQ] Pipe exported data directly to file

### ISSUE THREAD

**Issue by ghost** [sequelpro/sequelpro#1060]
Issue imported from Google Code: https://code.google.com/p/sequel-pro/issues/detail?id=1060
Reporter:hamiltont Date:2011-05-19 22:49:43 Status:WontFix Closed:2011-07-01

Perhaps I just can't find this feature, other 'export' related feature requests mention that it exists, so either it does and I can't find it or it got removed from my version. 
(see <a href="http://code.google.com/p/sequel-pro/issues/detail?id=1002&amp;can=1&amp;q=export%20type%3Aenhancement%20status%3Afixed%2Cverified%2Cimplemented&amp;colspec=ID%20Type%20Status%20Priority%20Milestone%20Owner%20Summary%20Stars" rel="nofollow">http://code.google.com/p/sequel-pro/issues/detail?id=1002&amp;can=1&amp;q=export%20type%3Aenhancement%20status%3Afixed%2Cverified%2Cimplemented&amp;colspec=ID%20Type%20Status%20Priority%20Milestone%20Owner%20Summary%20Stars</a> )

Basically, when I have a custom query that will generate a massive amount of data, I would like to be able to pipe it to a file, rather than show it in the data viewer, and then choose file-&gt;export-&gt;custom query result

I'm using version 0.9.8.1, build 2492, and MySql 5.1.34
